apps/discovery_api/test/unit/discovery_api/data/mapper_test.exs

```diff
@@ -28,7 +28,7 @@ defmodule DiscoveryApi.Data.MapperTest do
         [%{business: %{conformsToUri: nil}}, [:conformsToUri], "https://project-open-data.cio.gov/v1.1/schema/"],
         [%{business: %{conformsToUri: "overridden"}}, [:conformsToUri], "https://project-open-data.cio.gov/v1.1/schema/"],
         [%{technical: %{sourceFormat: "application/json"}}, [:fileTypes], ["JSON"]],
-        [%{technical: %{sourceFormat: "application/gtfs"}}, [:fileTypes], ["JSON"]],
+        [%{technical: %{sourceFormat: "application/gtfs+protobuf"}}, [:fileTypes], ["GTFS"]],
```

Comment:
I would expect fileTypes here to have "JSON", not GTFS. I believe this was covered with an override somewhere in the bit that generates the file types.